CMakeLists.txt

```diff
@@ -19,6 +19,11 @@ cmake_minimum_required(VERSION 3.17)
 
 project(Video_Processing_Framework)
 
+if(WIN32)
+    include_directories(${CUDA_TOOLKIT_ROOT_DIR}/include)
+    link_directories(${CUDA_TOOLKIT_ROOT_DIR}/lib/x64)
+endif(WIN32)
```

Comment:
this shouldn't be added manually and without `PRIVATE`/`PUBLIC` specifier. See #397, where I would use https://cmake.org/cmake/help/latest/module/FindCUDAToolkit.html

Reply:
changes from local repo got pushed, did not intend to do it.